src/arch/xtensa/boot_loader.c

```diff
@@ -276,11 +277,25 @@ static int32_t lp_sram_init(void)
 }
 #endif
 
+#if defined(CONFIG_TRACE_UART)
```

Comment:
Please start using #if (CONFIG_*) instead of _defined_, like in recent changes in sof (#if defined CONFIG_ *-> #if CONFIG_*)